bookkeeper-server/src/main/java/org/apache/bookkeeper/util/OrderedSafeExecutor.java

```diff
@@ -31,6 +31,7 @@
 import java.util.concurrent.ScheduledThreadPoolExecutor;
 import java.util.concurrent.ThreadFactory;
 import java.util.concurrent.TimeUnit;
+import org.apache.bookkeeper.client.BKException;
```

Comment:
why do you import this?

Reply:
it was a leftover of the first implementation.
dropped